src/plugins/plugins.h

```diff
@@ -223,6 +223,25 @@ inline timeval get_time()
 #define FORMAT_TIMEVAL "%" PRId64 ".%06" PRId64
 #define UNPACK_TIMEVAL(t) (t).tv_sec, (t).tv_usec
 
+inline void print_injection_info(output_format_t format, vmi_pid_t pid, const char* file)
```

Comment:
This should probably be in `libinjector.h`, not in `plugins.h`.

Reply:
Ok